README (2).md

```diff
@@ -0,0 +1,27 @@
+# P3Frontend
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.2.4.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+## Code scaffolding
+
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+
+## Build
+
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```

Comment:
why is this file re-added? i guess it's a little useful but this looks more like an oversight.

Reply:
Corrected, just had to pick one.

Reply:
Initially, we kept both so that we could let the devs see what the previous devs had.

On a closer look, we saw that one looked outdated, so we scrapped it and kept the one that looked newer.